app/forms/gpo_verify_form.rb

```diff
@@ -6,18 +6,24 @@ class GpoVerifyForm
   validate :validate_otp_not_expired
   validate :validate_pending_profile
 
-  attr_accessor :otp, :pii_attributes
+  attr_accessor :otp, :pii, :pii_attributes
   attr_reader :user
 
-  def initialize(user:, otp: nil)
+  def initialize(user:, pii:, otp: nil)
     @user = user
+    @pii = pii
     @otp = otp
   end
 
   def submit
     result = valid?
     if result
       if pending_in_person_enrollment?
+        UspsInPersonProofing::EnrollmentHelper.new.save_in_person_enrollment(
+          user,
+          user.pending_profile,
```

Comment:
Looks like there's a memoized method for this in the class already.


```suggestion
          pending_profile,
```

Reply:
Should we use it below on line 27 too?

Reply:
not a blocker